docs/runbooks/CDIDefaultStorageClassDegraded.md

```diff
@@ -2,12 +2,16 @@
 
 ## Meaning
 
-This alert fires when there is no default storage class that supports smart
-cloning (CSI or snapshot-based) or the ReadWriteMany access mode.
+This alert fires when there is/are default (Kubernetes and/or virtualization)
+storage class(es), but none of them supports both smart cloning (CSI or snapshot
+based) and ReadWriteMany access mode.
```

Comment:
Lots of parentheses :) We try to avoid them entirely in the downstream docs, so please remove or reduce them here. I can see your intention to clarify the "no default storage class" part, but I might do something like this instead:

```
This alert fires if the default storage class does not support smart
cloning (CSI or snapshot-based) or the ReadWriteMany access mode.
```

If you think we need to specify the multiple storage classes part, maybe:

```
This alert fires if the default storage class does not support smart
cloning (CSI or snapshot-based) or the ReadWriteMany access mode. The 
alert does not fire if at least one default storage class supports 
these features.
```